app/collections/preview.cjsx

```diff
@@ -28,17 +28,18 @@ module.exports = React.createClass
       @setState {subjects}
 
   render: ->
-    [owner, name] = @props.collection.slug.split('/')
-
+    baseLink = ""
+    if @props.project?
```

Comment:
Actually, this is a required prop, isn't it?  In other words, this is only rendered in the context of a project I think